OLCNE/Vagrantfile

```diff
@@ -105,7 +105,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   # Use specific component version (mainly used for development)
   OLCNE_VERSION = default_s('OLCNE_VERSION', '')
   K8S_VERSION = default_s('K8S_VERSION', '')
```

Comment:
To be consistent, we can drop the K8S_VERSION from the Vagrantfile as well

Reply:
Yep, good catch.